samples/dotnet/graph-api-skills/Program.cs

```diff
@@ -137,7 +137,7 @@ public static async Task Main()
             throw new InvalidOperationException("Unable to determine current assembly directory.");
         }
 
-        string skillParentDirectory = Path.GetFullPath(Path.Combine(currentAssemblyDirectory, "../../../../skills"));
+        string skillParentDirectory = Path.GetFullPath(Path.Combine(currentAssemblyDirectory, "../../../../../skills"));
```

Comment:
I think this might be due to the compilation settings and where the code ends up running. What about re-using `RepoUtils.SampleSkillsPath()`? see https://github.com/microsoft/semantic-kernel/blob/main/samples/dotnet/kernel-syntax-examples/RepoUtils/RepoFiles.cs

Reply:
Thanks, I will try that!

Reply:
That works. Had to make RepoFiles and the method public, hope that is OK. Also had to add namespace to Program.cs file to avoid type collision.